### DISCUSSION TITLE
Next scheduled meeting (November 1st) is holiday in many countries

### DISCUSSION THREAD

**Discussion by jlurien** [camaraproject/QualityOnDemand#375] [General]
We didn't discuss during last meeting that next one is national holiday in many parts of the world (as in Spain). Not sure how many people will be able to join.

**Comment by eric-murray** [comment]
I can attend, but happy to skip this one

**Reply by RandyLevensalor** [reply]
> I can attend, but happy to skip this one

Same here. Not a holiday for the US, but happy to skip if this impacts several others.

**Comment by hdamker** [comment]
I won't be able to attend - due to the public holiday and private reasons.

**Comment by RandyLevensalor** [comment]
I think that we should cancel the meeting for this Friday.

No one has come forward with a reason that we need to meet this Friday.  We would be missing several key contributors as well.

**Comment by RandyLevensalor** [comment]
The meeting has been canceled.
